packages/manager/apps/web/client/app/domain/list/template.html

```diff
@@ -142,6 +142,23 @@
     <oui-datagrid-column
         data-title="::'domains_column_domains_name_server' | translate"
     ></oui-datagrid-column>
+
+    <oui-action-menu text="Actions" compact>
+        <oui-action-menu-item
+            on-click="$ctrl.goToRestoreRenew($row)"
+            aria-label="{{:: 'domains_action_per_line_renew' | translate }}"
+            data-ng-if="$ctrl.constructor.isDomainRenewableOrRestorable($row)"
+        >
+            <span
```

Comment:
Hello, can't you only put one span with a conditional translate string in a ternary?